motif-docs/components/Badge.mdx

```diff
@@ -0,0 +1,9 @@
+import { Badge } from 'https://cdn.motif.land/motifblocks@0.0.4'
```

Comment:
Everything from `motif-docs` is a sync. You can ignore all of it.
To see real changes, check first and last commit only.